src/main/java/com/minecolonies/tileentities/ScarecrowTileEntity.java

```diff
@@ -0,0 +1,91 @@
+package com.minecolonies.tileentities;
+
+import java.util.Random;
+import com.minecolonies.inventory.InventoryField;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.tileentity.TileEntity;
+
+/**
+ * The scarecrow tile entity to store extra data.
+ */
+public class ScarecrowTileEntity extends TileEntity
+{
+    /**
+     * NBTTag to store the type.
+     */
+    private static final String TAG_TYPE = "type";
+
+    /**
+     * The name of the inventory which shows up when opened.
+     */
+    private static final String INVENTORY_NAME = "Scarecrow";
+    //todo change to .lang term
+
+    /**
+     * The inventory connected with the scarecrow.
+     */
+    public InventoryField inventoryField;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make inventoryField a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)